app/src/shared/modules/user/services.js

```diff
@@ -70,3 +70,10 @@ export const logout = (): Promise<any> => (
     Promise.resolve('')
 )
 
+export const deleteUserAccount = (): Promise<null> => (
+    new Promise((resolve, reject) => {
+        setTimeout(() => {
+            reject(new Error('Deleting user account is not supported yet!'))
```

Comment:
I wonder if we should put in some kind of searchable flag for user-exposed TODO items so we don't forget to either turn them off or implement them before release.  Maybe something like: https://eslint.org/docs/rules/no-warning-comments